meshtastic/deviceonly.proto

```diff
@@ -86,23 +87,23 @@ message NodeInfoLite {
   /*
    * local channel index we heard that node on. Only populated if its not the default channel.
    */
-  uint32 channel = 7;
+  optional uint32 channel = 7;
 
   /*
    * True if we witnessed the node over MQTT instead of LoRA transport
    */
-  bool via_mqtt = 8;
+  optional bool via_mqtt = 8;
```

Comment:
Not sure if this and the other bool are worth it, but it'd let 2.5+ nodes denote affirmatively that they did _not_ see a node via MQTT/a node is not a favorite. hops_away and channel I discussed in the main PR description, though.